maintenance/wikia/WikiFactoryVariables/migrateWikiWordmarks.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/**
+* Maintenance script to manage WikiFactory variable for one wiki or multiple wikis (from file)
+* - enable/disable the extension (set to true/false)
+* - set value to the variable (only string, integer, boolean types)
+* - remove variable from the wiki
+* - get the variable value
+* This is one time use script
+* @usage
+* 	# this will migrate wordmark-image-url for wiki with ID 119:
+* 	migrateWikiWordmarks -d --wikiId 119 --verbose --keyName wordmark-image-url
```

Comment:
I think due to changes in parameters it should be something like `/usr/wikia/backend/bin/run_maintenance '--script=migrateWikiWordmarks.php -d --verbose --keyName wordmark-image-url' --id=119`

Reply:
fixed: 25fee815f6e